utils/api/configurations.js

```diff
@@ -148,6 +148,32 @@ export const configureMessages = (data) => {
   }))
 }
 
+export const  configureFiles = (data) => {
+  // filter out the notes that do not have attachments 
+  const notesWithFiles = data.filter(d => d.attachments?.length !== 0)
+  let fileArrays = []
+
+  // modify the object for each file
+  notesWithFiles.map(note => {
+    let singleFileArray = note.attachments.map(file => ({
+      contentLength: formatBytes(file.content_length),
+      contentType: file.content_type,
+      createdAt: normalizeDate(file.created_at),
+      download: file.download,
+      fileName: file.filename,
+      href: `https://${process.env.NEXT_PUBLIC_PROVIDER_NAME}.scientist.com/secure_attachments/${file.uuid}`,
+      status: note.status,
```

Comment:
the corresponding wcl pr has `status: null` as a default prop. if the status may not exist, it should be handled here instead.

Reply:
Hmmm. I'm a little confused what you mean. So should it be 
`status: note.status || null` or are you saying i should make up a default status string to give the ones that are null?

because the status is coming in from the marketplace as null, not undefined. Its not that it doesn't exist- they all have a status field, its just that some are null/allowed to be null.

I was wondering about this in the PropTypes in the wcl too and looked it up, and this is what i found. Theres not really a proptype that allows a nullable field: https://stackoverflow.com/questions/37842868/how-can-i-declare-a-proptype-corresponding-to-a-nullable-number

But it also doesn't seem right to change the status from the marketplace to something other than null.

Reply:
I reviewed the wcl pr first and thought that you were setting the status to null, not keeping it at null. 

if https://github.com/scientist-softserv/webstore-component-library/pull/150#discussion_r1100509889 works, then this line can remain as is. otherwise, yes, I'd say we change the value to `status: note.status || ''`. then the proptype can always expect a string. plus, since an empty string is still falsy, we maintain the...intent, of the null value, for our purposes at least.